lxd/instance.go

```diff
@@ -380,6 +382,8 @@ func instanceCreateAsCopy(s *state.State, opts instanceCreateAsCopyOpts, op *ope
 		}
 	}
 
+	inst.LocalConfig()["volatile.uuid"] = uuid.New()
```

Comment:
Changing volatile.uuid in a commit that says " instance: volatile.uuid remain unchanged for snapshot/backup restores " doesn't make sense to me im afraid.